packages/yourloops/pages/signup/signup-account-form.tsx

```diff
@@ -71,30 +67,27 @@ function SignUpAccountForm(props: SignUpFormProps): JSX.Element {
   const { state, dispatch } = useSignUpFormState();
   const { handleBack, handleNext } = props;
 
+  const [hasPasswordError, setHasPasswordError] = React.useState(false);
+  const [password, setPassword] = React.useState("");
   const [username, setUsername] = React.useState("");
-  const [newPassword, setNewPassword] = React.useState("");
-  const [confirmNewPassword, setConfirmNewPassword] = React.useState("");
   const [inProgress, setInProgress] = React.useState(false);
   const [usernameTextFieldFocused, setUsernameTextFieldFocused] = React.useState(false);
 
-  const passwordCheck = checkPasswordStrength(newPassword);
-
   const errors: Errors = React.useMemo(
     () => ({
       username: _.isEmpty(username.trim()) || !REGEX_EMAIL.test(username),
```

Comment:
We could gradually remove lodash for useless methods
```suggestion
      username: !username.trim() || !REGEX_EMAIL.test(username),
```